src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
     return this._disabled;
   }
 
+  /**
+   * Indicated whether to hide the flag in the input element.
+   * @default "false"
+   */
+  @Input()
+  public hideSelectedCountryFlag: boolean;
+
+  /**
+   * Indicated whether to include phone information in the selected country and country dropdown.
```

Comment:
Is this supposed to be `Indicated`? Or should it be `Indicates`?

Reply:
It was a typo. Fixed.